CHANGELOG.md

```diff
@@ -9,7 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - N/A
 
 ### Changed
-- N/A
+- `RetryFuture` no longer requires the error type to implement `Drop`.
```

Comment:
```suggestion
- `RetryFuture` no longer requires the error type to implement `Display`.
```

Reply:
Gah, guess Drop was still on my mind from trying to rephrase the message. :facepalm: